app/packs/src/decidim/vocdoni/admin/steps/vote.js

```diff
@@ -1,41 +1,67 @@
-const ELECTION_VOTES_SELECTOR = ".js-votes-count";
-// Wait for 30s between stats refresh
+// Interval time in milliseconds for refreshing the statistics
 const WAIT_TIME_MS = 30000;
 
+// Fetch the votes from the API
 export const getElectionResults = async () => {
-  const vocdoniClientMetadata = document.querySelector(".js-vocdoni-client");
-  const resultsPath = vocdoniClientMetadata.dataset.resultsPath;
-  const response = await fetch(resultsPath);
-  const result = await response.json();
-  return result;
+  const vocdoniClientElement = document.querySelector(".js-vocdoni-client");
+  if (vocdoniClientElement) {
+    const resultsPath = vocdoniClientElement.dataset.resultsPath;
+    if (resultsPath) {
+      const response = await fetch(resultsPath);
+      return response.json();
+    }
```

Comment:
These two condictionals can be refactored to use a guard clause. I think it'd be cleaner